src/leap/bitmask/services/mail/conductor.py

```diff
@@ -84,11 +81,11 @@ def _handle_imap_events(self, req):
         :param req: Request type
         :type req: leap.common.events.events_pb2.SignalRequest
```

Comment:
According to the API changes in `leap.common.events`, this method should  be something like this:

```python
    def _handle_imap_events(self, event, content):
        """
        Callback handler for the IMAP events

        :param event: The event that triggered the callback.
        :type event: str
        :param content: The content of the event.
        :type content: list
        """
        if event == catalog.IMAP_SERVICE_STARTED:
            self._on_imap_connected()
        elif event == catalog.IMAP_SERVICE_FAILED_TO_START:
            self._on_imap_failed()
        elif event == catalog.IMAP_CLIENT_LOGIN:
            self._on_mail_client_logged_in()
```